templates/theme/.npm/build-favicons.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var svg2favicons = require('svg2favicons');
+var log = require('npmlog');
+
+try {
+  svg2favicons({
+    src: '../images/icons/favicon/favicon.svg',
+    dest: '../images/icons/favicon',
+    icoPath: '../favicon.ico',
+
+    developer: 'Adapt A/S',
+    developerURL: 'http://adapt.eu/',
+
+    // Everything below is project specific
+    appName: null,           // `string`
```

Comment:
Please make sure these gets processed using the twig rendere so that as much as possible gets prefilled.